test/unit/test_itde_manager.py

```diff
@@ -57,6 +57,8 @@ def test_bring_itde_up(mock_spawn_env, secrets, env_info):
         nameserver=(NAME_SERVER_ADDRESS,),
         db_mem_size="4 GiB",
         db_disk_size="10 GiB",
+        # this version has to be aligned with workflow setting
+        docker_db_image_version="8.31.0",
```

Comment:
Maybe use a constant

Reply:
This value comes from github workflow, so it is not possible to reuse the constant in both places